ui/static/ui/js/learning_resources.jsx

```diff
@@ -123,6 +124,7 @@ define('learning_resources', [
         terms: terms,
         description: this.state.description
       };
+      $(this.getDOMNode()).find('.lr-spinner').css('display', 'inline');
```

Comment:
You must not mess with the DOM in React. The only exceptions occur in certain lifecycle functions and this doesn't apply. You have to alter the state here and have React rerender with the spinner enabled or disabled.